### ISSUE TITLE
Improve Keyboarding for My Company Sections

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#10172]
This area needs improvement, starting by defining an accessible design pattern, tabbing order and focus management.

Although this is currently "tabbable" for a sighted user (they will need to tab A LOT though) the hearing-experience sequence won't make any sense for Screen Reader users.

**Example:**
The selected cost center (in red) is open on the right side, if users want to interact with the right-side panel they would have to tab all they down until the focus reaches the panel.

![Screen Shot 2020-12-01 at 6 55 41 PM](https://user-images.githubusercontent.com/15113219/100810730-1401a680-3407-11eb-9314-7fa1cf97485d.png)

One suggestion is fitting either an ARIA accordion or a tab panel design pattern, where the user navigates the left-side list with the arrow keys, activates with spacebar or enter, and tabs to jump into the selected panel.

- https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
- https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html

**Comment by developpeurweb** [comment]
`a11y-e2e-test` should be updated for this area as part of the effort, as mentioned by @giancorderoortiz in ticket #10081 not all tests seem to be covered by E2E.
